bin/node/node/Cargo.toml

```diff
@@ -15,6 +15,7 @@ futures = "0.3.1"
 log = "0.4.8"
 structopt = "0.3.8"
 bridge-node-runtime = { version = "0.1.0", path = "../runtime" }
+sp-bridge-eth-poa = { version = "0.1.0", path = "../../../primitives/ethereum-poa" }
```

Comment:
I don't think we should name crates `sp-`. It stands for "Substrate Primitives", and that's actually Bridge Primitive and not Substrate Primitive :)

Reply:
Yeap, I was thinking about that actually. Same with the leading `pallet-` since that's Substrate specific.